{{cookiecutter.repo_name}}/dev-requirements.txt

```diff
@@ -2,39 +2,34 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --extra=dev --output-file='..\..\dev-requirements.txt' '..\..\pyproject.toml'
+#    pip-compile --extra=dev --output-file=dev-requirements.txt pyproject.toml
 #
 black==22.1.0
-    # via newchess (..\..\pyproject.toml)
+    # via template-python-test (pyproject.toml)
```

Comment:
I think you've tested the cookiecutter template with template-python-test  as your project name?   

May we should remove the requirements.txt and requirements-dev.txt from the template  and use a post-generation hook for cookiecutter ?   https://cookiecutter.readthedocs.io/en/1.7.2/advanced/hooks.html

Reply:
Done!

I added a post-hook shell script which is linux-only. cookiecutter mentions there is a .bat integration as well but I am unable to test it. Could you perhaps test and add it?